game/game.gd

```diff
@@ -1,12 +1,24 @@
 extends Node2D
 
-signal on_power_change(new_power_state: bool)
-
+@export var generator_power:int = 100
+signal on_generator_power_change(new_level: int)
+signal on_power_changed(power_state: bool)
```

Comment:
I would call them something that explains better what it represents, like
- `on_power_level_changed`
- `on_power_state_changed`
It keeps it consistent with the emitted variables.